Bojzen-I-Mitten/Stort-spelprojekt
# Refactor/master

System Improvements:
+ Scene Management (load/create/destroy)
+ Object handler
+ Object & Component Destruction
+ Shader Management
+ Assembly loading (Project)
+ More

Req. Testing:
+ Verify object creation/destruction works
+ Verify reloading engine reload functions work
+ Verify set/unset static works
